### DISCUSSION TITLE
WTH Dyson Core Integration

### DISCUSSION THREAD

**Discussion by Anto79-ops** [libdyson-wg/ha-dyson#208] [General]
Please vote:

https://community.home-assistant.io/t/wth-make-dyson-core-integration/805058

**Comment by dotvezz** [comment]
Oh this would be so cool.

**Reply by dotvezz** [reply]
Okay maybe not :sweat_smile: https://community.home-assistant.io/t/wth-make-dyson-core-integration/805058/3

**Reply by Anto79-ops** [reply]
Lol, at least we made some noise .

**Reply by dotvezz** [reply]
It was fun while it lasted :rofl:

**Reply by mundschenk-at** [reply]
I mean you could just make PR, there is no formal process to get approval beforehand. @shenxn [started one](https://github.com/home-assistant/core/pull/53801), but I guess didn't have time to follow-up and @Kakise withdrew it.
